Description: Revert 0ac3bef72473c619194514d01ca55f2ed8c617c3
 Revert this commit for now, until we have a newer libinput
Author: Constantin Piber <cp.piber@gmail.com>

Index: source/src/managers/input/InputManager.cpp
===================================================================
--- source.orig/src/managers/input/InputManager.cpp
+++ source/src/managers/input/InputManager.cpp
@@ -1240,11 +1240,6 @@ void CInputManager::setPointerConfigs()
                     libinput_device_config_scroll_set_natural_scroll_enabled(LIBINPUTDEV, g_pConfigManager->getDeviceInt(devname, "natural_scroll", "input:natural_scroll"));
             }
 
-            if (libinput_device_config_3fg_drag_get_finger_count(LIBINPUTDEV) >= 3) {
-                const auto DRAG_3FG_STATE = sc<libinput_config_3fg_drag_state>(g_pConfigManager->getDeviceInt(devname, "drag_3fg", "input:touchpad:drag_3fg"));
-                libinput_device_config_3fg_drag_set_enabled(LIBINPUTDEV, DRAG_3FG_STATE);
-            }
-
             if (libinput_device_config_dwt_is_available(LIBINPUTDEV)) {
                 const auto DWT = sc<enum libinput_config_dwt_state>(g_pConfigManager->getDeviceInt(devname, "disable_while_typing", "input:touchpad:disable_while_typing") != 0);
                 libinput_device_config_dwt_set_enabled(LIBINPUTDEV, DWT);
Index: source/CMakeLists.txt
===================================================================
--- source.orig/CMakeLists.txt
+++ source/CMakeLists.txt
@@ -146,7 +146,7 @@ pkg_check_modules(
   pixman-1
   xcursor
   libdrm
-  libinput>=1.28
+  libinput
   gbm
   gio-2.0
   re2)
